drivers/clock_control/clock_stm32l4_wb.c

```diff
@@ -41,9 +41,6 @@ void config_enable_default_clocks(void)
 #ifdef CONFIG_CLOCK_STM32_LSE
 	/* LSE belongs to the back-up domain, enable access.*/
 
-	/* Enable the power interface clock */
-	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
```

Comment:
This is not undefined for L4.
So it would be preferred to restrict use under #idef LL_APB1_GRP1_PERIPH_PWR

Reply:
Oh, thanks. I've missed that this file is shared between L4 and WB. done